### DISCUSSION TITLE
I made a resourcepack for create mod

### DISCUSSION THREAD

**Discussion by F371X-1079** [tr7zw/ItemSwapper#65] [Show and tell]
Repo: https://github.com/F371X-1079/itemswapper-create/tree/main
Resource pack:
[itemswapper-create.zip](https://github.com/tr7zw/ItemSwapper/files/10179377/itemswapper-create.zip)

**Comment by F371X-1079** [comment]
it will be uploaded soon because i just figured it out!

**Reply by F371X-1079** [reply]
https://github.com/F371X-1079/itemswapper-create/tree/main

**Reply by tr7zw** [reply]
Can you put the zipped-up file onto the top post, so people have an easy time downloading it?

**Reply by F371X-1079** [reply]
sure just gotta figure it out

**Reply by tr7zw** [reply]
I think you can just drag it on there.
![grafik](https://user-images.githubusercontent.com/5133936/206275402-3f8de9b8-df2d-4548-915a-a46760513afe.png)

**Reply by F371X-1079** [reply]
well i googled it and i cant do it the that way it just doesnt let me pull it into the comment box

**Reply by F371X-1079** [reply]
> Can you put the zipped-up file onto the top post, so people have an easy time downloading it?

so because it doesnt work sadly this has to do for now :/

**Reply by tr7zw** [reply]
Updated the post, I was able to just paste in the zip after allowing it in NoScript.

**Reply by F371X-1079** [reply]
oh dear seems like i still have to figure out alot
hehe

**Reply by AdvancedWipe** [reply]
@F371X-1079 let me suggest you, to add a proper license to your repository. So people have rules what they can do. Since you write that, ItemSwapper can integrate it. You might want to use one that is compatible with the one ItemSwapper is using.

**Reply by F371X-1079** [reply]
> @F371X-1079 let me suggest you, to add a proper license to your repository. So people have rules what they can do. Since you write that, ItemSwapper can integrate it. You might want to use one that is compatible with the one ItemSwapper is using.

so i should use the same license they use?

**Reply by tr7zw** [reply]
The best/intended case would be, that mods like create can include the few pallet files on their end, so that we don't dictate how things have to be grouped. The alternative (for now) will be this show-and-tell section, where people can show and distribute their packs.

**Reply by F371X-1079** [reply]
> The best/intended case would be, that mods like create can include the few pallet files on their end, so that we don't dictate how things have to be grouped. The alternative (for now) will be this show-and-tell section, where people can show and distribute their packs.

that makes more sense i mean then they can then add stuff to their pallet when they update too
